sw/airborne/arch/chibios/halconf.h

```diff
@@ -62,6 +69,14 @@
 #define HAL_USE_CAN                 FALSE
```

Comment:
would be nice to add #if USE_CAN1 | USE_CAN2 as a first step towards using CAN in ChibiOS